.husky/_/husky.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+if [ -z "$husky_skip_init" ]; then
```

Comment:
Did you commit this by mistake? I think if you remove this and then do an `npm i` it will go away